__tests__/categories.test.js

```diff
@@ -73,12 +69,53 @@ describe('GET request for reviews,', function () {
                         comment_count: expect.any(String) //
                     });
                 });
+                expect(res.body.reviews).toBeSortedBy('created_at', { descending: true })
             });
     });
+
     test('Returns 404', function () {
         return request(app).get('/api/reviewz').expect(404);
     });
-});
-
 
+});
 
+describe('GET request for specific review ID', function () {
+    test('Returns 200', function () {
+        return request(app)
+            .get('/api/reviews/1').expect(200)
+            .then(function (res) {
+                expect(res.body.message).toEqual(('ALL OK'))
+            })
```

Comment:
I probably wouldn't bother about sending any extra info on the response body other than the data itself. The 200 status code is telling us all we need to know about the situation being all okay 🙂